client/src/axios-auth.js

```diff
@@ -89,7 +89,14 @@ instance.interceptors.response.use(function (response) {
     $(error.config.spinner).addClass('hidden')
   }
 
-  if (error.request && error.request.responseURL && error.request.responseURL.includes('/api/v1/events/')) {
+  if (
+    error.request && error.request.responseURL && 
+    (
+      error.request.responseURL.includes('/api/v1/events/') ||
+      error.request.responseURL.includes('api/v1/corporations') && 
+      error.response.data.message=="Error: Could not find corporation details"
```

Comment:
Thanks, much clearer 👍 

I don't know why the PR checks aren't running automatically.

Please run `npm run lint`. I think it will complain about the mix of || and && above. I don't even know from memory which operators have precedence here. Is it left to right? Or does && take precedence over || (same as boolean multiplication vs addition)???

Reply:
I don't find lint errors in axios-auth.js, even though there are over 2000 lint errors in this project.

Reply:
> I think it will complain about the mix of || and && above.

Reply:
I see that this project does not have vue-cli-service and therefore you cannot run lint locally.

I still see the following problems above:
1.  What is your intended operator precendence (ie, "a || b && c") ? It's unclear to me how this should work. Please add parentheses.
2. Line 97 should use `===` (and also add spaces around it).

Reply:
Is there document for the coding style so that I can following at the first time of code change?

Reply:
No, but a lot of these things are handled by the default + custom lint rules.

Eg, https://github.com/bcgov/business-create-ui/blob/main/.eslintrc.js